### PR TITLE
Update Authors

### PR DESCRIPTION
**Pull request by CDiaz-MS**
## Overview

- Update the author list for 2.7!

### CODE REVIEW

File: Authors.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Authors
+Authors
```
Comment (CDiaz-MS) [inline Authors.md:1]:
```suggestion
# Authors
```

File: Authors.md [hunk 2]
```diff
@@ -7,10 +7,9 @@ The Microsoft Mixed Reality Toolkit is a collaborative project containing contri
 - Addison Linville (radicalad)
 - ADP-JoNeff
 - afarchy
-- Against Lightning XR (againstlightning)
+- Against Lightning (AgainstLightning)
 - alandergrouse (alandergrouse)
 - Alex Floyd (elbuhofantasma)
-- Alexander Seeck (Alexees)
```
Comment (keveleigh) [inline Authors.md:13]: I think this removal (and the SGHolospace one below) happens when a user removes the email address they used to commit from their GitHub account, so it's no longer linked. I wonder if we should try to maintain the entries?

**Review by keveleigh — APPROVED** [review]

**Review by davidkline-ms — APPROVED** [review]

File: Authors.md (hunk 2 above)
Comment (CDiaz-MS) [reply Authors.md:13]: I can add re-add the names for now, but we can update the contributors script for next time

File: Authors.md [hunk 3]
```diff
@@ -7,10 +7,9 @@ The Microsoft Mixed Reality Toolkit is a collaborative project containing contri
 - Addison Linville (radicalad)
 - ADP-JoNeff
 - afarchy
-- Against Lightning XR (againstlightning)
+- Against Lightning (AgainstLightning)
 - alandergrouse (alandergrouse)
 - Alex Floyd (elbuhofantasma)
```
Comment (CDiaz-MS) [inline Authors.md:12]:
```suggestion
- Alex Floyd (elbuhofantasma)
- Alexander Seeck (Alexees)
```

File: Authors.md [hunk 4]
```diff
@@ -103,15 +106,17 @@ The Microsoft Mixed Reality Toolkit is a collaborative project containing contri
...
+- Philipp (AllBecomesGood)
 - ritijain
 - Robert Onulak (Ziugy)
 - Roberto Sonnino (robertos)
 - RogPodge
 - Roland Smeenk (rolandsmeenk)
 - Rosthouse (Rosthouse)
 - rwinj
+- SabinMGeorge
 - Sarah (SarahSexton)
```
Comment (CDiaz-MS) [inline Authors.md:119]:
```suggestion
- Sarah (SarahSexton)
- SGHolospace
```

File: Authors.md (hunk 2 above)
Comment (CDiaz-MS) [reply Authors.md:13]: I think we should maintain the names on the list

**Pull request merged**